### PR TITLE
Migration: Normalize release contract fields

### PR DESCRIPTION
**Pull request by joshbwlng**
Change-type: patch
Signed-off-by: Josh Bowling <josh@monarci.com>

### CODE REVIEW

**Comment by joshbwlng** [comment]
@fisehara How should async migrations be handled for `open-balena-api`? It differs from `balena-api` as we know when we can open a subsequent PR to finish up and stop async migrations. Perhaps we should instead not use async migrations here? I'm unsure.

**Comment by joshbwlng** [comment]
@balena-ci rebase

**Comment by joshbwlng** [comment]
@balena-ci rebase

File: src/migrations/0074-normalize-release-contract.async.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    `,
+	syncSql: `\
+    UPDATE "release" SET "contract" = "release"."contract"::jsonb#>>'{}' WHERE id IN (
+        SELECT "id" FROM "release" WHERE STARTS_WITH("contract", '"') FOR UPDATE SKIP LOCKED
+    );
+    `,
+	delayMS: 10000,
+	backoffDelayMS: 4000,
+	errorThreshold: 15,
+	asyncBatchSize: 1000,
```
Comment (joshbwlng) [inline src/migrations/0074-normalize-release-contract.async.ts:19]: Trying to be somewhat conservative with 1000 records every 10 seconds.

**Review by fisehara — APPROVED** [review]
LGTM

Although I think 1k rows in 10 seconds is very conservative.

**Review by Page- — APPROVED** [review]

**Pull request merged**